vnext/Scripts/rnw-dependencies.ps1

```diff
@@ -9,6 +9,9 @@ param(
     [String[]]$Tags = @('appDev')
 )
 
+# CODESYNC \packages\@react-native-windows\cli\src\runWindows\runWindows.ts
+$MarkerFile = "$env:LOCALAPPDATA\rnw-dependencies.txt"
```

Comment:
nit: it seems that the convention is to use lower case letters to start variable names.